source/renderer/app/stores/SidebarStore.js

```diff
@@ -62,6 +80,50 @@ export default class SidebarStore extends Store {
     });
   }
 
+  _sortWallets = (wallets: Wallet[]): Wallet[] => {
```

Comment:
This could be moved out of the store into a unit-testable helper function. That would reduce the store code also simplify the specs since you wouldn't have to set up the store for each case